.gitpod.yml

```diff
@@ -5,10 +5,12 @@ ports:
       onOpen: open-preview
 tasks:
     - init: >
+        gem install bundler &&
+        bundle config set without 'production' &&
+        bundle install &&
```

Comment:
some one of these lines isn't running; upon boot, i can run `bundle install` -- bundler is installed, but apparently not the gems.

Reply:
All subsequent steps seem to run; at least we have config.yml, database.yml, and schema.rb...

Reply:
Where can we find the output of the gitpod `init` steps?

Reply:
For reference, this works: 

https://github.com/publiclab/mapknitter/blob/641f6d2b61bf2dfa89cd6a5b96a82d0d3bbb2729/.gitpod.yml#L9-L18